airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -12,7 +12,7 @@ java {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.40.7'
+    cdkVersionRequired = '0.40.10'
     features = ['db-sources', 'datastore-postgres']
     useLocalCdk = false
```

Comment:
Do we want to ask CI to run against this PR once more by flipping this to true before we merge?